packages/lesswrong/lib/collections/sequences/schema.ts

```diff
@@ -150,15 +159,8 @@ const schema: SchemaType<"Sequences"> = {
     canUpdate: ['admins', 'sunshineRegiment'],
     canCreate: ['admins', 'sunshineRegiment'],
     ...schemaDefaultValue(false),
-  },
-
-  hideFromAuthorPage: {
-    type: Boolean,
-    optional: true,
-    canRead: ['guests'],
-    canUpdate: ['members'],
-    canCreate: ['members'],
-    ...schemaDefaultValue(false),
+    group: formGroups.adminOptions,
+    tooltip: "Hidden sequences don't show up on lists/search results on this site, but can still be accessed directly by anyone",
```

Comment:
This seems to be what this field is for, but I'm not 100% sure so let me know if it's wrong